jupyter_releaser/cli.py

```diff
@@ -289,9 +289,19 @@ def main(force):
     click.option(
         "--pydist-check-cmd",
         envvar="RH_PYDIST_CHECK_CMD",
-        default="pipx run twine check --strict",
+        default="pipx run twine check --strict {dist_file}",
         help="The command to use to check a python distribution file",
     ),
+    click.option(
+        "--pydist-extra-check-cmds",
+        envvar="RH_EXTRA_PYDIST_CHECK_CMDS",
+        default=[
+            "pipx run 'validate-pyproject[all]' pyproject.toml",
```

Comment:
How does this affect projects that use a root `pyproject.toml` for configuration and other metadata?

It looks like this might be a bit too strict in some cases. Is there a way to skip that check via the releaser config?

An example run where this affects the check release workflow: https://github.com/jupyterlite/jupyterlite/actions/runs/3764123821/jobs/6398242276

Reply:
This also seems to have an effect on JupyterLab, although probably good it catches potential issue there:

https://github.com/jupyterlab/jupyterlab/actions/runs/3764228169/jobs/6398454511

Reply:
It is targeting the `pyproject.toml` in the checkout dir.  Here is an example config for [`ipykernel`](https://github.com/ipython/ipykernel/blob/9b434e91e3182a55cc041e924e9b55da92947c2d/pyproject.toml#L265).

Reply:
Nice thanks for the example :+1: